src/Parser.js

```diff
@@ -165,7 +165,7 @@ class Parser {
             if (breakOnInfix && functions[lex.text] && functions[lex.text].infix) {
                 break;
             }
-            const atom = this.parseAtom();
+            const atom = this.parseAtom(body);
```

Comment:
This is maybe a little hacky, but some commands need to know what was the previous atom in the current group.